### PR TITLE
New glasses sprites and inhands

### PR DESCRIPTION
**Pull request by alsoandanswer**
what it says on the tin.

also cleans inhand files. again.

https://forums.aurorastation.org/topic/13621-new-glasses-sprites/

made a feedback thread

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

**Comment by alsoandanswer** [comment]
!review

File: code/modules/clothing/glasses/glasses.dm [hunk 1]
```diff
@@ -65,13 +71,18 @@ BLIND     // can't see anything
 			to_chat(usr, "You activate the optical matrix on the [src].")
 			if(activated_color)
 				set_light(2, 0.4, activated_color)
-		user.update_action_buttons()
+	user.update_action_buttons()
+	if(ismob(loc))
+		var/mob/living/M = loc
```
Comment (NonQueueingMatt) [inline code/modules/clothing/glasses/glasses.dm]: Why are you using loc here?

**Comment by alsoandanswer** [comment]
!review

**Comment by NonQueueingMatt** [comment]
Again, not a fan of the aviator sprite. The HUD showing makes it far worse.

Also, aren't the gas masks inhands actually used?

**Comment by alsoandanswer** [comment]
> Again, not a fan of the aviator sprite. The HUD showing makes it far worse.
> 
> Also, aren't the gas masks inhands actually used?



> Again, not a fan of the aviator sprite. The HUD showing makes it far worse.
> 
> Also, aren't the gas masks inhands actually used?

For the onmobs, they are practically unnoticable coupled with other stuff,
however, for the ontable sprite itself, I'd have to agree, but I feel like it is useful, tangible gameplay-wise as it shows the HUD thingy as a tangible thing which gives it the super cool sci-fi scanny goggles effect.

Also, they're color coded, so you can actually tell apart the security shades from the engineer shades at a glance and vice versa.

also gas mask inhands have been moved to their own file, i did that in my masks update but didn't actually remove them

**Comment by alsoandanswer** [comment]
!review

**Comment by alsoandanswer** [comment]
!review

**Comment by alsoandanswer** [comment]
!review

**Comment by alsoandanswer** [comment]
!review

**Pull request closed without merging**